rsts/index.rst

```diff
@@ -31,12 +31,12 @@
    concepts/architecture
 
 .. toctree::
-   :caption: Deployment
+   :caption: Administrator's Guide
    :maxdepth: -1
-   :name: deploymenttoc
+   :name: administratortoc
    :hidden:
 
-   deployment/index
+   administrator/index
```

Comment:
Can we consider keeping the `Deployment` name here?

I think we want to keep a product/functionality-based nomenclature in the docs as opposed to a role-based nomenclature, because the latter alienates users who don't play that role and discourages them from exploring that section of the documentation.

We want to help educate people who don't necessarily have a background in infra to be able to potentially set up their own Flyte cluster. Of course we can talk about _who_ the pages are primarily for, but I think we should let that be part of the introduction to this overall section of the docs.

Happy to discuss alternatives to "Deployment", e.g. "Cluster Deployment", etc.

Reply:
Yeah I guess that's fine... as long as what we mean by that is all things that relate to a deployment of Flyte.  Because the initial deployment of Flyte is just one component of it.  There are things related to maintaining, scaling, plugins, etc.  All the observability and monitoring stuff should also go here.  Upgrade procedures should go here, etc.